client/layout/masterbar/logged-out.jsx

```diff
@@ -31,7 +31,7 @@ const MasterbarLoggedOut = ( { title, sectionName, translate } ) => (
 			}
 
 			{ 'login' !== sectionName
-			?	<Item url={ config( 'login_url' ) }>
+			?	<Item url={ config( 'login_url' ) + '?redirect_to=' + encodeURIComponent( window.location.href ) }>
```

Comment:
@astralbodies Since wp-desktop is not using `wp-login.php` (at least from what I can see), this redirection will not work. I'm not sure if this will be a problem (at least the extra parameter doesn't seem to be causing any problems), but maybe there's a corresponding parameter we could use on desktop?